src/components/Requirements.vue

```diff
@@ -1,6 +1,13 @@
 <template v-if="semesters">
   <div class="requirements">
-    <div class="fixed">
+    <div class="fixed"
+      data-intro='<b>This is your Requirements Bar 📋</b><br>
```

Comment:
Same thing with this emoji

Reply:
Already mentioned the issue on slack, but I'm not sure how to add the built filepath to the data-intro string